pkg/networkmanager/firewall.jsx

```diff
@@ -324,13 +324,13 @@ export class Firewall extends React.Component {
             addServiceAction = (
                 <OverlayTrigger className="pull-right" placement="top"
                                 overlay={ <Tooltip id="tip-auth">{ _("You are not authorized to modify the firewall.") }</Tooltip> } >
-                    <Button bsStyle="primary" className="pull-right" disabled> {_("Add Services…")} </Button>
+                    <Button bsStyle="primary" className="pull-right" disabled> {_("Add Services")} </Button>
```

Comment:
Because users assume that something happens for a button anyway, and thus it's redundant? This breaks translations, but okay.

Reply:
We don't have dots anywhere else, for example in networking we have `Add Bond`, `Add Team`...